nntrainer/layers/preprocess_translate_layer.cpp

```diff
@@ -45,75 +47,100 @@ int PreprocessTranslateLayer::initialize(Manager &manager) {
     affine_transform_mat.at<float>(1, 1) = 1;
 
     input_mat =
-      cv::Mat::zeros(input_dim[0].height(), input_dim[0].width(), CV_32FC3);
+      cv::Mat::zeros(input_dim_0.height(), input_dim_0.width(), CV_32FC3);
     output_mat =
-      cv::Mat::zeros(input_dim[0].height(), input_dim[0].width(), CV_32FC3);
+      cv::Mat::zeros(input_dim_0.height(), input_dim_0.width(), CV_32FC3);
```

Comment:
This is not related with this PR, but add todo here ( implementation without opencv )

Reply:
I will add.

Reply:
done.